fetcher/src/job/job.controller.ts

```diff
@@ -54,6 +69,12 @@ export class JobController {
       throw new HttpException(msg, HttpStatus.NOT_FOUND)
     }
 
+    if (aggregator.fetcherType != BigInt(FETCHER_TYPE)) {
```

Comment:
If we change the type of fetcher type inside of prisma, we can remove this `BigInt` type here.